src/app/cube/about-us/components/mission/mission.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="mission-container">
+  <div class="left-side">
+    <img class = "mission-logo" src="../../../../../assets/images/logo.png" alt="Logo of CLARK Man">  
+  </div>
+  <div class="right-side">
+    <div class="text-container"> 
+      <h2>At Clark, we... the mission statement will go here in place of the words that are currently here. More stuff here</h2>
```

Comment:
@pzalep1 Do you know if there is a mission statement written yet?  Otherwise, maybe about us should be a dark deploy.

Reply:
I believe Dr. Kaza has one. We can ask at stand up.